docs/tasks/setup_all_or_nothing.md

```diff
@@ -0,0 +1,456 @@
+# All-or-nothing scheduling for jobs
```

Comment:
I don't want to hype the expectations of users by mentioning "all-or-nothing" in the title.

Do you have other suggestions?

Reply:
- Setup pod scheduling tracking
- Setup serial admission for guaranteed scheduling
?

Reply:
I chose the `Sequential Admission with Ready Pods` suggestion.